scan/src/components/TxIndexPageTable.re

```diff
@@ -238,15 +238,15 @@ let renderEditOracleScript = (oracleScript: TxSub.Msg.EditOracleScript.t) => {
         <AddressRender address={oracleScript.owner} />
       </div>
     </div>
-    <VSpacing size=Spacing.lg />
-    <div className=Styles.topicContainer>
-      <Text value="NAME" size=Text.Sm weight=Text.Thin spacing={Text.Em(0.06)} />
-      <div className=Styles.hFlex>
-        <TypeID.OracleScript id={oracleScript.id} />
-        <HSpacing size=Spacing.sm />
-        <Text value={oracleScript.name} />
-      </div>
-    </div>
+    // <VSpacing size=Spacing.lg />
+    // <div className=Styles.topicContainer>
+    //   <Text value="NAME" size=Text.Sm weight=Text.Thin spacing={Text.Em(0.06)} />
+    //   <div className=Styles.hFlex>
+    //     <TypeID.OracleScript id={oracleScript.id} />
+    //     <HSpacing size=Spacing.sm />
+    //     <Text value={oracleScript.name} />
+    //   </div>
+    // </div>
```

Comment:
Please remove comments
If you want to leave comments, please add reason with todo